ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Hey team! 
+
+I was using your awesome geocoding engine when I noticed something interesting.
+Let me tell you more about it.
+
+---
+#### Here's what I did :innocent:
+_try adding a link here to either [pelias.io/compare](http://pelias.io/compare) or [mapzen.com/search](https://mapzen.com/search)_
```

Comment:
this might be better as a comment, like used in the [npm issue template](https://github.com/npm/npm/blob/master/.github/issue_template.md). But i'm not really sure. I just fear we would see this exact line as the only content in this section for most of the issue reports